fpmms/costing/costing.py

```diff
@@ -0,0 +1,197 @@
+import tiktoken
+import anthropic
+from typing import Optional
+from .prompts import prediction_request
+from .prompts import prediction_request_sme
+from .prompts import prediction_request_claude
```

Comment:
We keep a local hardcoded copy. Not ideal if the prompts change in the future. On the other hand, these tools are pretty much finalized so it should not be an issue.

What do you think @DavidMinarsch?

Reply:
I'd prefer that we imported the https://github.com/valory-xyz/mech repo as a subrepo and then we avoid the tool code duplication. @richardblythman can you do that?

Reply:
I think subrepo is a better option. Will work on this

Reply:
We did it this way before (with a symlink). Do you want me to change it back, and add some details to the readme for creating the symlink? Or using a git submodule? @DavidMinarsch @Adamantios

Reply:
submodule please